frontend/app/styles/colors-autogen.css

```diff
@@ -35,6 +35,7 @@
 .fill-light-blue-bg { fill: $light-blue-bg }
 .fill-white { fill: $white }
 .fill-borderColor { fill: $borderColor }
+.fill-figmaColors { fill: $figmaColors }
```

Comment:
not sure what's happened here since 'figmacolors' is an object inside theme/colors

Reply:
Rebase issue I think. Removed all.